csrf/mod.ts

```diff
@@ -13,13 +13,30 @@ const primaryToken = createHash("sha512");
 primaryToken.update(v4.generate());
 const primaryTokenString = primaryToken.toString();
 
-export function CSRF() {
+type Options = {
+  cookie?: boolean;
+};
+
+const defaultOptions = {
+  cookie: false,
+};
+
+export function CSRF(options?: Options) {
+  if (!options) {
+    options = defaultOptions;
+  }
   const csrf = <F> function csrf(
     request: Drash.Http.Request,
     response?: Drash.Http.Response,
   ): void {
     if (response) {
-      const requestToken = request.headers.get("X-CSRF-TOKEN");
+      let requestToken: string | null = "";
+
+      if (options!.cookie === true) {
+        requestToken = request.getCookie("X-CSRF-TOKEN");
+      } else {
+        requestToken = request.headers.get("X-CSRF-TOKEN");
+      }
```

Comment:
Since we already set a default value via parameter, we can ditch the `!` from the conditional.
Final implementation would be something like:
```typescript
if (options.cookie === true) {
  requestToken = request.getCookie("X-CSRF-TOKEN");
} else {
    requestToken = request.headers.get("X-CSRF-TOKEN");
}
```

Reply:
Yeah sweet, the above comment means we can do that now

Reply:
https://github.com/drashland/deno-drash-middleware/pull/57/commits/0285f7db1ab835b58c616b0de5c44524cd7b4dec